presets/presets.go

```diff
@@ -543,7 +552,9 @@ func (b *Builder) createMenus(ctx *web.EventContext) (r h.HTMLComponent) {
 		menus = append(menus, b.menuItem(ctx, m, false))
 	}
 
-	r = VList(menus...).Class("primary--text")
+	r = h.Div(
+		VList(menus...).Class("primary--text").Dense(true).Nav(true),
```

Comment:
Hi @moluuser .
I think that there is no need to add `.Nav(true)`. After adding it, the width of the menu name becomes smaller and the height becomes larger, which makes the gap between menus too large.
And it's not used anywhere (Listing tables, select items ...) else.

Reply:
@zealllot OK, thank you for your review.😄